build/KoreBuild.sh

```diff
@@ -14,7 +15,13 @@ __exec() {
 
     local cmdname=$(basename $cmd)
     echo -e "${CYAN}> $cmdname $@${RESET}"
-    $cmd "$@"
+
+    if [ -z "${TRAVIS}" ]; then
+        $cmd "$@"
+    else
+        # Work around https://github.com/Microsoft/msbuild/issues/1792
+        $cmd "$@" | tee /dev/null
```

Comment:
Add `set -o pipefail` to the top of file. Otherwise this will cause all commands to exit with exit code 0.